src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -1813,6 +1813,23 @@ def update_resource(cmd, parameters, resource_ids=None,
          for id_dict in parsed_ids])
 
 
+def patch_resource(cmd, resource_ids=None, resource_group_name=None,
+                   resource_provider_namespace=None, parent_resource_path=None, resource_type=None,
+                   resource_name=None, api_version=None, latest_include_preview=False):
+    parsed_ids = _get_parsed_resource_ids(resource_ids) or [_create_parsed_id(cmd.cli_ctx,
+                                                                              resource_group_name,
+                                                                              resource_provider_namespace,
+                                                                              parent_resource_path,
+                                                                              resource_type,
+                                                                              resource_name)]
+
+    parameters = {}
+    return _single_or_collection(
+        [_get_rsrc_util_from_parsed_id(cmd.cli_ctx, id_dict, api_version, latest_include_preview)
+         .patch(parameters) for id_dict in parsed_ids]
+    )
```

Comment:
May I ask how to pass in the parameters to be updated?

Reply:
Currenly updating parameters is not supported. I will add some new parameters to support.